architecture/Annotations.md

```diff
@@ -0,0 +1,49 @@
+# Annotations
+
+An annotation is a string that can be used to provide context to an action or motion. 
```

Comment:
Are we documenting this in the context of the CDapp ? or the contracts themselves ? 

Because technically annotations, as a function of the contracts, provide context to other transactions.

Reply:
I'm thinking in the context of the CDapp, and specifically, for a (potentially new) developer unfamiliar with the feature